src/main/java/net/snowflake/client/jdbc/SnowflakeDatabaseMetaData.java

```diff
@@ -2564,6 +2564,116 @@ public ResultSet getTypeInfo() throws SQLException {
         statement);
   }
 
+  /**
+   * Function to return a list of streams
+   *
+   * @param originalCatalog catalog name
```

Comment:
Just curious as to why we use "original", as it doesn't seem we are creating a new one.